tests/unit/tf/core/test_prediction.py

```diff
@@ -70,5 +70,5 @@ def flip_target(target):
     features, targets = mm.sample_batch(ecommerce_data, batch_size=100)
     outputs, context = model(features, targets=targets, training=True, output_context=True)
 
-    flipped = np.logical_not(targets["click"].numpy()).astype(np.int)
```

Comment:
Why is targets no longer a dict here? This feels like it should be unrelated to the dataloader change.

Reply:
Dataloader [no longer returns a dictionary for targets](https://github.com/NVIDIA-Merlin/dataloader/blob/498543d54629216dc09a5854dca4dbeeabcab356/merlin/dataloader/loader_base.py#L626-L631), and will pick one column and use it if there are multiple target columns. I'm not sure if this is what we want, but it is how dataloader behaves for now.

Reply:
Following team discussion, it looks like having dataloader output a tensor, not a dictionary, might not work for Models. We might need to change the dataloader's behavior upstream.